cudamapper/src/utils.cpp

```diff
@@ -156,13 +156,13 @@ static inline void encode_seq(bam1_t* const alignment, const std::string& seq)
             15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15,
             15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15, 15};
 
-    const size_t len = seq.length();
+    const int32_t len = get_size(seq);
```

Comment:
Copying over from previous PR: @mimaric fixed to use signed_integer_util functions